question-service/app/crud/questions.py

```diff
@@ -28,3 +28,28 @@ async def get_all_questions() -> QuestionCollection:
 async def get_question_by_id(question_id: str) -> QuestionModel:
     existing_question = await question_collection.find_one({"_id": ObjectId(question_id)})
     return existing_question
+
+async def delete_question(question_id: str):
+    existing_question = await question_collection.find_one({"_id": ObjectId(question_id)})
+    if existing_question is None:
+        return None
+    await question_collection.delete_one({"_id": ObjectId(question_id)})
+    return {"message": f"Question with id {existing_question['_id']} and title '{existing_question['title']}' deleted."}
+
+async def update_question_by_id(question_id: str, question_data: QuestionModel):
+    existing_question = await question_collection.find_one({"_id": ObjectId(question_id)})
+    if existing_question is None:
+        return None
+    
+    updated_data = {
```

Comment:
Is it possible to have a model for this as well?

Reply:
Will do!